packit_service/worker/helpers/build/babysit.py

```diff
@@ -155,7 +157,12 @@ def update_testing_farm_run(event: TestingFarmResultsEvent, run: TFTTestRunTarge
         )
         # check for identifiers equality
         if handler.pre_check(package_config, job_config, event_dict):
-            handler.run_job()
+            signatures.append(TestingFarmResultsHandler.get_signature(event=event, job=job_config))
```

Comment:
```suggestion
            signatures.append(handler.get_signature(event=event, job=job_config))
```

Reply:
No, it should be a method of the class object and not of its instance.